### ISSUE TITLE
Upgrade @slack/web-api once a fix is released because of axios vulnerability

### ISSUE THREAD

**Issue by krider2010** [krider2010/slack-bot-action#1]
Tracking issue in the @slack/web-api repo: https://github.com/slackapi/node-slack-sdk/issues/796

This was shown to be an issue in this Action from GitHub security vulnerability notices, and the following command line details show that axios exists because of the above explicit dependency.

```bash
→ npm ls axios
slack-bot-action@1.0.0 /path/to/krider2010/slack-bot-action
└─┬ @slack/web-api@5.0.0
  └── axios@0.18.0 
```

**Comment by anulman** [comment]
@krider2010 saw you were tracking the upstream issue; wanted to notify you that 0.18.1 was released. Fix instructions [here](https://github.com/slackapi/node-slack-sdk/issues/796#issuecomment-498068370)

**Comment by krider2010** [comment]
Really appreciate that @anulman - I'll get it sorted today 🙇‍♀

**Comment by krider2010** [comment]
Fixed by #2
